packages/data-explorer/package.json

```diff
@@ -24,7 +24,7 @@
     "@molgenis/molgenis-api-client": "^3.1.3",
     "@molgenis/molgenis-i18n-js": "^0.2.1",
     "@molgenis/molgenis-ui-context": "^1.3.4",
-    "@molgenis/molgenis-ui-filter": "^1.1.2",
+    "@molgenis/molgenis-ui-filter": "^1.4.0",
```

Comment:
This is now 1.4.2 and how does this relate to this PR: https://github.com/molgenis/molgenis-frontend/pull/173

molgenis-ui-filter@1.4.2 contains: "activeFilter wil not brake when updated quickly"

Reply:
updated